backend/app/Http/Controllers/Security/ShowSecurityController.php

```diff
@@ -18,6 +20,18 @@ class ShowSecurityController extends Controller
      */
     public function __invoke(): View|Factory
     {
-        return view('security.index', []);
+        $userId = Auth::id();
+
+        /**
+         * usersテーブルからログインユーザーのレコードを取得
+         * @var User
+         * @property string $created_at
+         */
+        $loginUser = User::where('id', $userId)->first();
+
+        // 一応エラーを防ぐためcreated_atが存在しているときのみformatをかける
+        $registerDate = $loginUser->created_at ? $loginUser->created_at->format('Y/n/j') : '-----';
+
+        return view('security.index', ['registerDate' => $registerDate]);
```

Comment:
パスワードはハッシュ化してるから表示できないけど、現状のメールアドレスはフォームに入れておきたいので、
```suggestion
        return view('security.index', ['user' => $user]);
```
として、blade側でメールアドレスや登録日取り出す形でも良いかも！”

Reply:
bladeでold()でメアドとpw取り出せるのでは？と思ったんですけどそれじゃマズい感じですか？

Reply:
slackにて失礼しました！！